modelcard.md

```diff
@@ -13,23 +14,29 @@ datasets:
 metrics:
 - {metric_0}  # Example: wer. Use metric id from https://hf.co/metrics
 
+# Optional. Add this if you want to encode your eval results in a structured way.
 model-index:
 - name: {model_id}
   results:
-  - task: 
-      type: {task_type}  # Required. Example: automatic-speech-recognition
-      name: {task_name}  # Optional. Example: Speech Recognition
+  - task:
+      type: {task_type}             # Required. Example: automatic-speech-recognition
+      name: {task_name}             # Optional. Example: Speech Recognition
     dataset:
-      type: {dataset_type}  # Required. Example: common_voice. Use dataset id from https://hf.co/datasets
-      name: {dataset_name}  # Optional. Example: Common Voice zh-CN
-      args: {arg_0}         # Optional. Example: zh-CN
+      type: {dataset_type}          # Required. Example: common_voice. Use dataset id from https://hf.co/datasets
+      name: {dataset_name}          # Required. Example: Common Voice (French)
+      config: {dataset_config}      # Optional. Example: fr
+      split: {dataset_split}        # Optional. Example: test
+      revision: {dataset_revision}  # Optional. Example: 5503434ddd753f426f4b38109466949a1217c2bb
+      args:
+        {arg_0}: {value_0}          # Optional. Additional arguments to `load_dataset()`. Example for wikipedia: language: en
+        {arg_1}: {value_1}          # Optional. Example for wikipedia: date: 20220301
     metrics:
-      - type: {metric_type}    # Required. Example: wer
-        value: {metric_value}  # Required. Example: 20.90
-        name: {metric_name}    # Optional. Example: Test WER
-        args: {arg_0}          # Optional. Example for BLEU: max_order
+      - type: {metric_type}         # Required. Example: wer. Use metric id from https://hf.co/metrics
+        value: {metric_value}       # Required. Example: 20.90
+        name: {metric_name}         # Optional. Example: Test WER
+        args: {arg_0}               # Optional. Example for BLEU: max_order
```

Comment:
Should this also be `{arg_0}: {value_0}`

Reply:
Good catch!